Lab1 Peer Review

chkboris opened the pull request:
@jeff-abe 

Hey, sorry I couldn't get this to you yesterday, windows decided 8pm was a great time to update for 5+ hours :\

chkboris committed:
Added Lab 1 Notebook

What it says on the tin

chkboris committed:
Problem 1 Initial draft

Rough draft for problem 1, needs log plot and explanations

chkboris committed:
Clean 1-3

Cleaned up problems 1 through 3, added logarithmic plots

chkboris committed:
Lab 1 Completed

Lab 1 complete, awaiting peer revision

jeff-abe commented:
Firstly I would add a link to the z-score table or even add a screenshot in. I think the html file is ultimately supposed to stand on its own, rather than referencing lab materials. With your exponential distribution question, I think that your example could be used as a discrete example, just because there wouldn't be a fractional day with rain, unless you would want to consider that, but then I would state that explicitly. Besides those two things the lab was solid, good job.

chkboris committed:
Implement peer review changes

Added an internal link to the z-score table. Also included a brief investigation into fractional rainy days.

chkboris committed:
Added html version of lab

Lab 1 report is named Lab 1 solution
